examples/fabric-admin/commands/fabric-sync/FabricSyncCommand.h

```diff
@@ -19,6 +19,16 @@
 #pragma once
 
 #include <commands/common/CHIPCommand.h>
+#include <commands/pairing/OpenCommissioningWindowCommand.h>
+#include <commands/pairing/PairingCommand.h>
+
+constexpr uint32_t kSetupPinCode               = 20202021;
+constexpr uint16_t kMaxCommandSize             = 64;
+constexpr uint16_t kDiscriminator              = 3840;
+constexpr uint16_t kWindowTimeout              = 300;
+constexpr uint16_t kIteration                  = 1000;
+constexpr uint16_t kAggragatorEndpointId       = 1;
```

Comment:
Where is kAggragatorEndpointId used?

Reply:
It is used in bridge management, removed from this PR